### ISSUE TITLE
asciidoc output from markdown input nested ordered lists lose nesting

### ISSUE THREAD

**Issue by cirosantilli** [jgm/pandoc#4362]
pandoc 2.1.1:

```
printf '1.  a\n    1. b' | ~/test/pandoc-2.1.1/bin/pandoc -f markdown -t asciidoc
```

Gives:

```
1.  a
1.  b
```

Instead of the expected:

```
. a
.. b
```

The HTML conversion is nested as expected for example:

```
printf '1.  a\n    1. b' | ~/test/pandoc-2.1.1/bin/pandoc -f markdown -t html
<ol type="1">
<li>a
<ol type="1">
<li>b</li>
</ol></li>
</ol>
```

**Comment by jgm** [comment]
Current output is
```
[arabic]
. a
[arabic]
.. b
```
so I guess this has been fixed!

**Comment by cirosantilli** [comment]
Thanks, fix confirmed at 35971495abbc15342f0035040f964ed4dd3b0c5d.
